docs/streams/developer-guide/config-streams.html

```diff
@@ -91,14 +91,12 @@
               <li><a class="reference internal" href="#rocksdb-config-setter" id="id20">rocksdb.config.setter</a></li>
               <li><a class="reference internal" href="#state-dir" id="id14">state.dir</a></li>
               <li><a class="reference internal" href="#topology-optimization" id="id31">topology.optimization</a></li>
-              <li><a class="reference internal" href="#windowed-inner-class-serde" id="id38">windowed.inner.class.serde</a></li>
```

Comment:
The KIP deprecates this config. Is it pre-mature to remove it? Would it be better to just highlight it's deprecated now as we do for other configs?